src/types/redis_bitmap.cc

```diff
@@ -246,19 +246,20 @@ rocksdb::Status Bitmap::BitCount(const Slice &user_key, int64_t start, int64_t s
   uint32_t start_index = u_start / kBitmapSegmentBytes;
   uint32_t stop_index = u_stop / kBitmapSegmentBytes;
   // Don't use multi get to prevent large range query, and take too much memory
-  std::string value;
+  rocksdb::PinnableSlice pin_value;
```

Comment:
We should move this line into for-loop? so that we don't need to manually call the pin_value.Reset.

Reply:
Yes you're right. Reset() cost more time than creating for-loop local vars.